src/schemas/json/workflows.json

```
@@ -259,6 +259,48 @@
         },
         "for": {
           "$ref": "#/definitions/for"
+        },
+        "parallel": {
+          "type": "object",
+          "description": "Run branches or iterations in parallel",
+          "additionalProperties": false,
+          "properties": {
+            "exception_policy": {
+              "description": "determines action of other branches when an exception occurs",
+              "type": "string",
+              "enum": ["continueAll"]
+            },
+            "shared": {
+              "description": "list of shared variables",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "concurrency_limit": {
+              "type": "integer",
+              "description": "upper limit for branches/iterations to perform concurrently"
+            },
+            "branches": {
+              "description": "a list of branches that will run concurrently",
+              "type": "array",
+              "minItems": 2,
+              "maxItems": 10,
+              "items": {
+                "type": "object",
+                "properties": {
+                  "steps": {
+                    "description": "A list of steps.",
```

```suggestion
                    "description": "A list of steps",
```

Optional fix: sentence end consistency violated: trailing dot in not necessary.